core/domain/event_services_test.py

```diff
@@ -105,6 +109,124 @@ def test_record_solution_hit_events(self):
         self.assertEqual(model.time_spent_in_state_secs, 2.0)
 
 
+class StartExplorationEventHandlerTests(test_utils.GenericTestBase):
+
+    def test_recording_exploration_start_events(self):
+
+        all_models = (
+            stats_models.StartExplorationEventLogEntryModel.get_all())
+        self.assertEqual(all_models.count(), 0)
+
+        event_services.StartExplorationEventHandler.record(
+            'exp_id', 1, 'state_name', 'session_id',
+            {}, feconf.PLAY_TYPE_NORMAL)
+
+        all_models = (
+            stats_models.StartExplorationEventLogEntryModel.get_all())
+
```

Comment:
Perhaps drop this newline, so that this reads similarly to lines 116-118.

Ditto elsewhere.

Reply:
I removed this newline everywhere in the file to match the pattern.